src/groups/mqb/mqbs/mqbs_filestore.h

```diff
@@ -329,7 +329,8 @@ class FileStore : public DataStore {
 
     NodeReceiptContexts d_nodes;
 
-    DataStoreRecordKey d_lastRecoveredMessage;
+    bsls::Types::Uint64 d_lastStrongConsistencySequenceNum;
```

Comment:
Why not call it `DataStoreRecordKey d_lastRecoveredStrongConsistencyMessage`?

It shows clearly that the sequence number and the primary lease id are always used in conjunction.

Reply:
d